demo/hyproof-demo.md

```diff
@@ -0,0 +1,183 @@
+# HyProof Demonstration
+
+This document walks you through the steps required to deliver a complete demonstration of certificate issuance and revocation using the HyProof system
+
+## Basic Demo Prerequisites
+
+You will require `git` and `docker` to be installed on your machine in order to run these locally, and have some familiarity with a command line interface.
+
+In order to be able to reproduce the steps described in this document, you need to have both the back-end API (`sqnc-hyproof-api`) and front-end user interface (`sqnc-hyproof-client`) components running.
+
+The first step is to fetch both of these codebases from Github.
```

Comment:
My spellchecker is suggesting code base or code-base .. i think the spellchecker is wrong isn't it?

Reply:
Seems all are acceptable: https://english.stackexchange.com/questions/237835/codebase-or-code-base